zproject/urls.py

```diff
@@ -421,10 +421,12 @@
     path('export/realm', rest_dispatch,
          {'POST': 'zerver.views.realm_export.export_realm',
           'GET': 'zerver.views.realm_export.get_realm_exports'}),
-    re_path(r'^export/realm/(?P<export_id>.*)$', rest_dispatch,
-            {'DELETE': 'zerver.views.realm_export.delete_realm_export'}),
+    path('export/realm/<int:export_id>', rest_dispatch,
+         {'DELETE': 'zerver.views.realm_export.delete_realm_export'}),
 ]
 
+integrations = IntegrationView.as_view()
```

Comment:
Maybe these should be called e.g. `integrations_view` for clarity?

Reply:
Changed.